### ISSUE TITLE
Call generate_bindings from CMakeList.txt broken. generate_bindings must have two parameters

### ISSUE THREAD

**Issue by Rogni** [godotengine/godot-cpp#472]
https://github.com/godotengine/godot-cpp/blob/c9a740be34438ce999402b7b76304a38daaaa811/CMakeLists.txt#L135

https://github.com/godotengine/godot-cpp/blob/c9a740be34438ce999402b7b76304a38daaaa811/binding_generator.py#L16

```
-- Generating Bindings
Traceback (most recent call last):
  File "<string>", line 1, in <module>
TypeError: generate_bindings() missing 1 required positional argument: 'use_template_get_node'
```

**Comment by Rogni** [comment]
https://github.com/godotengine/godot-cpp/pull/456

**Comment by vnen** [comment]
Should be fixed since #456 is merged now.
